src/App.elm

```diff
@@ -48,34 +47,14 @@ init _ =
 
 
 type Msg
-    = UpdateQuery String
-    | CloseDefinition Hash
-    | ToggleShowCode Hash
-    | ToggleExpandedNamespaceListing Hash
+    = ToggleExpandedNamespaceListing Hash
     | FetchSubNamespaceListingFinished Hash (Result Http.Error NamespaceListing)
     | FetchRootNamespaceListingFinished (Result Http.Error NamespaceListing)
 
 
 update : Msg -> Model -> ( Model, Cmd Msg )
 update msg model =
     case msg of
-        UpdateQuery query ->
-            ( { model | query = query }, Cmd.none )
-
-        CloseDefinition hash ->
-            let
-                nextOpenDefinitions =
-                    model.openDefinitions
-            in
-            ( { model | openDefinitions = nextOpenDefinitions }, Cmd.none )
-
-        ToggleShowCode hash ->
-            let
-                nextOpenDefinitions =
-                    model.openDefinitions
-            in
-            ( { model | openDefinitions = nextOpenDefinitions }, Cmd.none )
-
```

Comment:
This weren't being used right now and was a bit presumptuous of the future implementation.